conda_smithy/feedstock_content/ci_support/checkout_code.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+
```

Comment:
We would want to add the base branch here to help in our detection of merge conflicts. However, CircleCI does not yet provide a general way to get that [information]( https://discuss.circleci.com/t/add-environment-varible-circle-pr-base-branch/6189 ). Once they do, we can do something like [this]( https://github.com/conda-forge/staged-recipes/pull/1486/files#diff-f9e6910612efc2dbe7c3774b355f2d97R6 ).